docker-compose.yml

```diff
@@ -6,22 +6,24 @@ services:
             context: .
             dockerfile: Dockerfile-flask
         image: yaptide_flask
-        # mount data volume as /usr/local/app/yaptide/data in rw mode
-        volumes:
-            - data:/usr/local/app/yaptide/data:rw
         container_name: yaptide_flask
         environment:
             - FLASK_DEBUG=1
             - CELERY_BROKER_URL=redis://redis:6379/0
             - CELERY_RESULT_BACKEND=redis://redis:6379/0
-            - FLOWER_BASIC_AUTH=admin:password
             - CERT_AUTH_URL=${CERT_AUTH_URL-""}
             - KEYCLOAK_BASE_URL=${KEYCLOAK_BASE_URL-""}
             - KEYCLOAK_REALM=${KEYCLOAK_REALM-""}
             - BACKEND_EXTERNAL_URL=${BACKEND_EXTERNAL_URL-""}
+            - FLASK_SQLALCHEMY_DATABASE_URI=postgresql://${POSTGRES_USER:-yaptide_user}:${POSTGRES_PASSWORD:-yaptide_password}@postgresql:5432/${POSTGRES_DB:-yaptide_db}
+        healthcheck:
+            test: ["CMD-SHELL", "pg_isready -h $${POSTGRES_HOST:-postgresql} -p $${POSTGRES_PORT:-5432} -U $${POSTGRES_USER:-yaptide_user}"]
+            interval: 10s
+            timeout: 5s
```

Comment:
This probably should be moved to `postgresql` service

Reply:
You are correct, that was an naive version of setting a dependency on postgresql container. I will change it to `depends_on` with `condition` option as described here: https://docs.docker.com/compose/compose-file/05-services/#depends_on